src/api/util.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * Utility functions are collected here. These are functions that are used
+ * by the exported functions below. Some of them are themselves exported.
+ *
+ * @prettier
```

Comment:
This file is indented with tabs, which means that prettier has not been run on it. Either you forgot to run it before committing, or this @prettier tag here needs to be in a comment of its own up above, as it is in the other files.  Please reformat.